src/foundation/src/PDFsharp/src/PdfSharp/Pdf.Signatures/PdfSignatureHandler.cs

```diff
@@ -69,6 +70,21 @@ public PdfSignatureHandler(ISigner signer, PdfSignatureOptions options)
             this.Options = options;
         }
 
+        public PdfSignatureHandler(ISigner signer, PdfSignatureOptions options, byte[] timestampToken)
```

Comment:
I would consider using the 2-params constructor by adding a 3rd param (timestampToken) that would default to null if not used.
If using an explicit 3-params constructor with no default null value, I would then add a check and throws an ArgumentNullException if the value is null.